src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -66,6 +67,11 @@ private Workspace(WorkspaceDescription wsmObject) {
     this.userFacingId = wsmObject.getUserFacingId();
     this.name = wsmObject.getDisplayName() == null ? "" : wsmObject.getDisplayName();
     this.description = wsmObject.getDescription() == null ? "" : wsmObject.getDescription();
+    if (wsmObject.getGcpContext() != null) {
+      this.cloudPlatform = CloudPlatform.GCP;
+    } else if (wsmObject.getAzureContext() != null) {
+      this.cloudPlatform = CloudPlatform.AZURE;
+    }
```

Comment:
do we need a default if it's neither?

Reply:
otherwise mark it @Nullable

Reply:
This class is the business object and not directly serialized. Hence the @nullable tag would not have any effect. 
the tag only lets the Json de/serializer know that if this field is missing in the serialized form, it can be set to Null instead of throwing an error. 

default value will be null, no tag needed.